vllm/attention/layer.py

```diff
@@ -912,23 +907,16 @@ def maybe_calc_kv_scales_fake(
 )
 
 
+@maybe_transfer_kv_layer
```

Comment:
@NickLucche or @ElizaWszola could one of you just check this is compatible with #25954?